core/nodejsActionBase/src/service.js

```diff
@@ -82,13 +99,15 @@ function NodeActionService(config) {
                     setStatus(Status.ready);
                     return responseMessage(200, { OK: true });
                 }).catch(function (error) {
-                    var errStr = error.stack ? String(error.stack) : error;
                     setStatus(Status.stopped);
-                    return Promise.reject(errorMessage(502, "Initialization has failed due to: " + errStr));
+                    var errStr = "Initialization has failed due to: " + error.stack ? String(error.stack) : error;
+                    return Promise.reject(errorMessage(502, errStr));
                 });
             } else {
                 setStatus(Status.ready);
-                return Promise.reject(errorMessage(403, "Missing main/no code to execute."));
+                var msg = "Missing main/no code to execute.";
+                console.error("Internal system error:", msg);
```

Comment:
this error will now appear is user's action logs - which is new behavior.

Reply:
dropped it